CalibCalorimetry/EcalTPGTools/plugins/EcalTPGParamBuilder.h

```diff
@@ -90,8 +90,8 @@ class EcalTPGParamBuilder : public edm::EDAnalyzer {
   std::string getDet(int tcc) ;
   std::pair < std::string, int > getCrate(int tcc) ;
 
-  const CaloSubdetectorGeometry * theEndcapGeometry_ ;
-  const CaloSubdetectorGeometry * theBarrelGeometry_ ;
+  CaloSubdetectorGeometry * theEndcapGeometry_ ;
```

Comment:
Why was the `const` removed?

Reply:
Chris - as soon as I put const for CaloSubdetectorGeometry* I get compilation error - I shall forward the error message to you.

Reply:
I shall send them in a separate mail to you